tests/plugins/range.test.ts

```diff
@@ -124,6 +124,53 @@ describe("range plugin", () => {
       });
     });
 
+    describe("create a range and remove multiple columns", () => {
+      beforeEach(() => {
+        m = new Model({
```

Comment:
model is already defined in the parent describe.
I know you copy/pasted, but the syntax in wrong, you can remove the `rows: 10, cols: 10`. They are 1) useless and 2) wrong, that's not how we define the number of cols/rows (see your other tests in `formula.test.ts`)